students/rimlee_talukdar/lesson01/assignment/inventory_management/electric_appliances_class.py

```diff
@@ -0,0 +1,27 @@
+"""Electric appliances class"""
+from inventory_management.inventory_class import Inventory
+
+class ElectricAppliances(Inventory):
+    """
+    Creates an instance of the ElectricAppliances Class with inheritance
+    from the Inventory class
+    """
+    def __init__(self, product_code, description,
+                 market_price, rental_price, brand, voltage):
+        # Creates common instance variables from the parent class
+        Inventory.__init__(self, product_code, description,
```

Comment:
Instead of specifying Inventory, since it is the parent, you can use super!

super().\_\_init\_\_(self, product_code, description, market_price, rental_price)